requirements-dev.txt

```diff
@@ -0,0 +1,4 @@
+pip>=9
+tox
+virtualenv
+histomicsui>=1
```

Comment:
`histomicsui` should be in the `install_requires` section of setup.py, not in `requirements-dev.txt`

Reply:
See cea46cd2c2d0a95d20b9c0acffd1cae51b6b056a